### ISSUE TITLE
supabase start: ApplyLayer exit status 1

### ISSUE THREAD

**Issue by akoenig** [supabase/cli#932]
# Bug report

## Describe the bug

When running `supabase start`, the following error occurs when pulling the `supabase/storage-api:v0.29.1` container image:

```sh
Error: failed to register layer: ApplyLayer exit status 1 stdout:  stderr: failed to Lchown "/usr/local/lib/node_modules/pm2/node_modules/nssocket/.npmignore" for UID 1389985163, GID 386085923 (try increasing the number of subordinate IDs in /etc/subuid and /etc/subgid): lchown /usr/local/lib/node_modules/pm2/node_modules/nssocket/.npmignore: invalid argument
```

I went a little deeper and it turned out, that the package [nssocket](https://www.npmjs.com/package/nssocket) uses a `uid` and `gid` for the file `.npmignore` which is (usually) not in the range configured in `/etc/subuid` and `/etc/subgid`.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. `supabase start`
2. See error

## Expected behavior

The local development environment should start.

## System information

- OS: A fresh `Ubuntu 22.04` based container
- Browser (if applies) -
- Version of supabase-js: -
- Version of Node.js: `18.15.0`

## Additional context

n/a

**Comment by sweatybridge** [comment]
Hi @akoenig are you running the CLI inside a docker container? If that's the case, you would also need to mount the docker socket as a volume into the container. Typically with `-v /var/run/docker.sock:/var/run/docker.sock` flag

**Comment by jeffreylouden** [comment]
I'm hitting the exact same error as @akoenig with `supabase/storage-api:v0.29.1`, but I am not running the CLI inside a docker container.

### System information

- OS: Fedora 37
- Browser (if applies) -
- Version of supabase-js: -
- Version of Node.js: 18.15.0

**Comment by jeffreylouden** [comment]
After banging my head against this for the past several weeks, I'm still hitting a brick wall. As a last-ditch effort, I completely overhauled my system (migrated to Fedora 37 to Pop_OS -- in case a debian-based distro had better luck), but my local development environment remains non-functional.

My clean system is stock Pop_OS 22.04 LTS running Docker in [rootless mode](https://docs.docker.com/engine/security/rootless/). 

My application isn't doing anything sophisticated and is a straightforwrd Remix application with Supabase. I encounter the originally posted error when running `npx supabase start` now on two different Linux distributions.

It's likely a limit of my Docker knowledge, but any suggestions for workarounds in lieu of a permanent fix would be hugely appreciated.

**Comment by sweatybridge** [comment]
I've opened a PR to fix the upstream storage-api docker image. Once that's merged, I will bump the image used by cli to address this issue. I'm not aware of any temp workarounds but I the permanent fix should be pretty fast.

Appreciate your patience.

**Comment by sweatybridge** [comment]
This has been fixed in CLI v1.50.8
